### PR TITLE
doc: added release note for map 21.10.1

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Release note for centreon map 21.10.1 (LOG4J included)

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [ ] 21.04.x
- [x] 21.10.x
- [ ] 22.04.x (master)

### CODE REVIEW

File: en/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -16,6 +16,27 @@ If you have feature requests or want to report a bug, please contact support.
 
 ## Centreon MAP
 
+### 21.10.1
+
+Release date: `December 16, 2021`
+
+#### Bug fixes
+
+- Path to map server backup log file in cron is now the proper path
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:25]:
```suggestion
- Path to map server backup log file in cron is now correct
```

File: en/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -16,6 +16,27 @@ If you have feature requests or want to report a bug, please contact support.
...
+- Simple link can now be created between a container and a view
+- Map server now supports metrics label with '=' and white spaces
+- Non-conventional metric labels do not cause map server to crash anymore
+- Configuration file for centreon map server is not replaced during upgrade anymore
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:29]:
```suggestion
- Configuration file for Centreon map server is not replaced during upgrade anymore
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: en/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -16,6 +16,27 @@ If you have feature requests or want to report a bug, please contact support.
...
 ## Centreon MAP
 
+### 21.10.1
+
+Release date: `December 16, 2021`
+
+#### Bug fixes
+
+- Path to map server backup log file in cron is now the proper path
+- Simple link can now be created between a container and a view
```
Comment (nmagniezCentreon) [inline en/releases/centreon-commercial-extensions.md:26]: Through API

File: fr/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -15,6 +15,27 @@ Pour faire des demandes d'évolutions ou reporter des bugs sur les extensions co
...
 ## Centreon MAP
 
+### 21.10.1
+
+Release date: `December 16, 2021`
+
+#### Bug fixes
+
+- Path to map server backup log file in cron is now correct
+- Simple link can now be created between a container and a view
```
Comment (cg-tw) [inline fr/releases/centreon-commercial-extensions.md:25]:
```suggestion
- A simple link can now be created between a container and a view
```

File: en/releases/centreon-commercial-extensions.md [hunk 5]
```diff
@@ -16,6 +16,27 @@ If you have feature requests or want to report a bug, please contact support.
...
 ## Centreon MAP
 
+### 21.10.1
+
+Release date: `December 16, 2021`
+
+#### Bug fixes
+
+- Path to map server backup log file in cron is now correct
+- Simple link can now be created between a container and a view
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:26]:
```suggestion
- A simple link can now be created between a container and a view
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**